.github/dependabot.yml

```
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  # Update Dockerfile
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "feat: "
```

I think increasing the patch version should be sufficient for most Dockerfile updates.

```suggestion
      prefix: "fix: "
```